doc/sphinx/proofs/writing-proofs/reasoning-inductives.rst

```diff
@@ -974,32 +974,42 @@ Generation of induction principles with ``Scheme``
    .. insertprodn scheme_kind sort_family
 
    .. prodn::
-      scheme_kind ::= Equality for @reference
-      | {| Induction | Minimality | Elimination | Case } for @reference Sort @sort_family
+      scheme_kind ::= {| Induction | Minimality | Elimination | Case } for @reference Sort @sort_family
       sort_family ::= Set
       | Prop
       | SProp
       | Type
 
-  A high-level tool for automatically generating
-  (possibly mutual) :term:`induction principles <induction principle>`
-  for given types and sorts.
-  Each :n:`@reference` is a different inductive type identifier belonging to
-  the same package of mutual inductive definitions.
-  The command generates the :n:`@ident`\s as mutually recursive
-  definitions. Each term :n:`@ident` proves a general principle of mutual
-  induction for objects in type :n:`@reference`.
+   Generates (possibly mutual) :term:`induction principles <induction principle>`
+   for given types and sorts. Each :n:`@reference` is a different inductive type
```

Comment:
```suggestion
   for given types and sorts. Each :n:`@reference` is an inductive type
```

Seems obvious to me you wouldn't repeat the same reference

Reply:
I'm not sure. Because you can generate different inductive principles, people might confuse it and try to generate `Induction`, `Minimality` etc. for different sorts all with the same Scheme command. Of course it won't work, but the `with` part is strictly about the mutual inductive type. The fact that the bodies might be well known but I don't think it hurts to keep it like this.